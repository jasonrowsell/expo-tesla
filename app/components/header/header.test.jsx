import React from 'react';
import { render } from '@testing-library/react-native';
import renderer from 'react-test-renderer';

import Header from './header';

describe('<Header />', () => {
  test('has 2 children', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree.children.length).toBe(2);
  });

  test('renders correctly', () => {
    const tree = render(<Header />);
    expect(tree).toMatchSnapshot();
  });
});
