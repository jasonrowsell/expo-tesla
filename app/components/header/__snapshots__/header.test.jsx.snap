// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders correctly 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "justifyContent": "space-between",
      "paddingHorizontal": 20,
      "position": "absolute",
      "top": 50,
      "width": "100%",
      "zIndex": 1,
    }
  }
>
  <Image
    source={1}
    style={
      Object {
        "height": 20,
        "resizeMode": "contain",
        "width": 100,
      }
    }
  />
  <Image
    source={1}
    style={
      Object {
        "height": 25,
        "resizeMode": "contain",
        "width": 25,
      }
    }
  />
</View>
`;
